action.yml

```diff
@@ -15,10 +15,10 @@ inputs:
   access_token:
     description: 'Your GitHub Access Token, defaults to: {{ github.token }}'
     default: '${{ github.token }}'
-    required: true
+    required: false
```

Comment:
It has a default value

Reply:
What about the `required` prop in `index.ts`?

Reply:
@styfle Should it be deleted?

Reply:
I'm not sure. Are either of these required props causing problems?

Reply:
They're not causing any problems to me, but I saw it's set to required although it has a default value so that's why I created this PR

Reply:
If its not causing problems, lets not change it 🙂

Reply:
I would argue that it also implies that the docs can be simplified, if this works without passing the default.